docs/reference/ilm/error-handling.asciidoc

```diff
@@ -70,24 +68,24 @@ Which returns the following information:
   "indices" : {
     "myindex" : {
       "index" : "myindex",
-      "managed" : true,                         <1>
-      "policy" : "shrink-the-index",            <2>
+      "managed" : true,                         
+      "policy" : "shrink-index",                <1>
       "lifecycle_date_millis" : 1541717265865,
-      "age": "5.1d",                            <3>
-      "phase" : "warm",                         <4>
+      "age": "5.1d",                            <2>
+      "phase" : "warm",                         <3>
       "phase_time_millis" : 1541717272601,
-      "action" : "shrink",                      <5>
+      "action" : "shrink",                      <4>
       "action_time_millis" : 1541717272601,
-      "step" : "ERROR",                         <6>
+      "step" : "ERROR",                         <5>
       "step_time_millis" : 1541717272688,
-      "failed_step" : "shrink",                 <7>
+      "failed_step" : "shrink",                 <6>
       "step_info" : {
-        "type" : "illegal_argument_exception",   <8>
-        "reason" : "the number of target shards [4] must be less that the number of source shards [2]" <9>
+        "type" : "illegal_argument_exception",  <7>
+        "reason" : "the number of target shards [4] must be less that the number of source shards [2]" <8>
```

Comment:
The associated number for this line (8) cannot be viewed unless you scroll the example. Is it worth adding a hard return after "...target shards" to wrap the line?
![image](https://user-images.githubusercontent.com/25848033/83654440-20599380-a58b-11ea-91f4-fb41b4f43f42.png)

Reply:
Fixed.